front-end/src/pages/ManageProject.js

```diff
@@ -1,20 +1,16 @@
-import React, { Component } from 'react';
+import React, {useEffect, useState} from 'react';
 import AddProject from "../components/AddProject";
 import ProjectsList from "../components/ProjectsList";
 
-class ManageProject extends Component {
 
-    render() {
-        return (
-          <div>
-              <h2> Add a new project: </h2>
-              <AddProject />
-              <br />
-              <h2> Select an existing project: </h2>
-              <ProjectsList />
-          </div>
-        );
-    }
+export default function ManageProject() {
```

Comment:
Good job changing ManageProject class component to a function as it is easier to use with hooks and is commonly used nowadays.